src/util/fees.cpp

```diff
@@ -26,16 +30,35 @@ std::string StringForFeeReason(FeeReason reason) {
     return reason_string->second;
 }
 
-bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
-    static const std::map<std::string, FeeEstimateMode> fee_modes = {
+const std::vector<std::pair<std::string, FeeEstimateMode>>& FeeModeMap() {
+    static const std::vector<std::pair<std::string, FeeEstimateMode>> fee_modes = {
         {"UNSET", FeeEstimateMode::UNSET},
         {"ECONOMICAL", FeeEstimateMode::ECONOMICAL},
         {"CONSERVATIVE", FeeEstimateMode::CONSERVATIVE},
+        {(CURRENCY_UNIT + "/KB"), FeeEstimateMode::BTC_KB},
+        {(ToUpper(CURRENCY_ATOM) + "/B"), FeeEstimateMode::SAT_B},
     };
-    auto mode = fee_modes.find(mode_string);
+    return fee_modes;
+}
 
-    if (mode == fee_modes.end()) return false;
+std::string FeeModes(const std::string& delimiter) {
+    std::ostringstream r;
+    bool first = true;
+    for (const auto& i : FeeModeMap()) {
+        r << (first ? "" : delimiter) + i.first;
+        first = false;
+    }
+    return r.str();
+}
 
-    fee_estimate_mode = mode->second;
-    return true;
+bool FeeModeFromString(const std::string& mode_string, FeeEstimateMode& fee_estimate_mode) {
+    auto searchkey = ToUpper(mode_string);
+    auto vector = FeeModeMap();
+    for (const auto& pair : vector) {
+        if (pair.first == searchkey) {
```

Comment:
Regarding https://github.com/bitcoin/bitcoin/pull/11413/files#r313929142

Then I'd do `ToUpper(pair.first) == searchkey)` and display the right casing.

Reply:
Done.